crates/sui-node/src/lib.rs

```diff
@@ -118,6 +119,19 @@ impl SuiNode {
             })
         };
 
+        let indexing_subsystem_handle = match index_store {
```

Comment:
we want to check `index_store || enable_event_processing` if we make event processing part of the subsystem